Assets/Scripts/Enemy.cs

```diff
@@ -0,0 +1,24 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Enemy : MonoBehaviour
+{
+    private float speed = 3.0f;
```

Comment:
unity style naming conventions says you should name your private variables with an underscore at the beginning. so _speed

Reply:
also you should add a [SerializedField] above the variable to let you modify it from the editor even though it is private